release_notes/3.2.2.md

```diff
@@ -1,6 +1,7 @@
-**Jira Release Notes - Published by Splunk December 22, 2021**
+**Jira Release Notes - Published by Splunk December 23, 2021**
 
 
-**Version 3.2.2 - Released December 22, 2021**
+**Version 3.2.2 - Released December 23, 2021**
 
-* Added "update_fields" and "time_spent" parameters in "set_status" action [PAPP-8959]
\ No newline at end of file
+* Added "update_fields" and "time_spent" parameters in "set_status" action [PAPP-8959]
+    * **Note**: We don't recommend using this connector version with SOAR v5.0.1 because of a known issue
```

Comment:
We don't recommend using this connector version with SOAR v5.0.1 because of a platform known issue [PSAAS-6036]

Reply:
done